.github/workflows/release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Release aws-auth binaries
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  releases-matrix:
+    name: Release Go Binary
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
```

Comment:
This is also at `v2`

Reply:
Fyi. You can setup `dependabot` to auto update the actions.

https://github.com/keikoproj/addon-manager/blob/master/.github/dependabot.yml